src/averageAges.js

```diff
@@ -15,11 +15,20 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredByMan = people.filter(person =>
+    century
+      ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+      : person.sex === 'm'
+  );
+
+  return getAvarageAge(filteredByMan);
+}
+
+function getAvarageAge(value) {
+  const sum
+  = value.reduce((result, man) => result + man.died - man.born, 0);
```

Comment:
use destructurizing structure throughout your code
```suggestion
  = value.reduce((result, ({ died, born })) => result + (died - born), 0);
```

Reply:
<img width="823" alt="Screenshot 2023-03-01 at 13 53 27" src="https://user-images.githubusercontent.com/117028255/222119503-db8f169e-da03-445e-ac63-b9f3c13a8ca9.png">

Reply:
Так не работает